wow_undercut.py

```
@@ -114,10 +163,24 @@ def create_embed(title, description, fields, color="red"):
     }
     return embed
 
+
 def split_list(input_list, max_length):
```

Ruff (0.8.2)

Missing return type annotation for public function `split_list`

(ANN201)